packages/lpsc/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=lpsc
```

Comment:
Could you please keep `local-php-security-checker` as pkgname but keep `lpsc` as binary name. Or rather keep `local-php-security-checker` as binary name but add `lpsc` as "alias" (not a true shell alias but a symlink) because it's always annoying to not be able to find the binary name without having to read the PKGBUILD for users.

So 

```
pkgname=local-php-security-checker
_pkgname=lscp
```

Reply:
how about naming both `pkgname` and the binary  `local-php-security-checker` as users wouldn't even know that `lpsc` existed when they install the package

Reply:
Yes in the end people can create their own aliases